src/mpyl/project.py

```diff
@@ -333,12 +333,16 @@ def from_config(values: dict):
 
 
 @dataclass(frozen=True)
-class S3Bucket:
-    bucket: TargetProperty[str]
+class CloudFrontS3Bucket:
+    name: TargetProperty[str]
+    region: str
 
     @staticmethod
     def from_config(values: dict):
-        return S3Bucket(bucket=TargetProperty.from_config(values.get("bucket", {})))
+        return CloudFrontS3Bucket(
+            name=TargetProperty.from_config(values.get("name", {})),
+            region=values.get("region", {}),
+        )
```

Comment:
To what extent is this bucket with specified region specific to Cloudfront?

In general I'm in favor of specific naming, but in this case I would tend toward referring to `s3` here, because 1. I think it's quite likely that this config will be used in other deployment scenarios involving s3 buckets, 2. the cloudfront deploy step already specifies the cloudfront specific use of s3 buckets, 3. MPyL is open source, so a (slightly) higher focus on reusability than with proprietary code would be justified.

Reply:
It's not specific to cloudfront but I was looking at the usages of the s3 config property and it's only being used for cloudfront so I thought it might be better to make that clear in the config.
I can revert the name back, I was just thinking it might be better to keep things specific until we have other known use cases

Reply:
If this were purely internal code I would agree, but in this case I would be in favor of reverting the name back to what it was. I don't think it's likely to lead to confusion and it saves us another schema change by the time we create other s3 related steps.

Reply:
I've reverted the naming back to s3